packages/manager/apps/telecom/src/app/telecom/pack/migration/translations/Messages_fr_FR.json

```diff
@@ -136,5 +136,11 @@
   "telecom_pack_migration_pto_yes_description": "Nous procéderons à l'écrasement de la prise optique (PTO) existante.",
   "telecom_pack_migration_pto_no_description": "Un technicien procèdera à l'installation d'une prise optique (PTO) lors de son intervention.",
   "telecom_pack_migration_pto_yes_not_known": "J'ai un boîtier mais je ne connais pas son numéro",
-  "telecom_pack_migration_pto_yes_not_known_description": "Avez-vous bien vérifié que vous ne disposez pas de la référence de la prise optique (PTO) ? Cela peut avoir avoir une incidence sur le délai de votre raccordement."
+  "telecom_pack_migration_pto_yes_not_known_description": "Avez-vous bien vérifié que vous ne disposez pas de la référence de la prise optique (PTO) ? Cela peut avoir avoir une incidence sur le délai de votre raccordement.",
+  "telecom_pack_migration_promotion_description": "Promotion disponible jusqu'au {{ endDate }}",
+  "telecom_pack_migration_promotion_subscription_months_offered": "{{ duration }} mois offerts",
+  "telecom_pack_migration_promotion_installation_fees_offered": "Frais de mises en service offerts",
+  "telecom_pack_migration_confirm_promotion": "Les frais d'installation et les {{ duration }} premiers mois de votre nouvel abonnement sont offerts (hors location de modem, options supplémentaires ou frais de transport).",
```

Comment:
What if `duration === 1` ?

Reply:
For the moment, there is only value for duration and is 3, I could write the message with only one value but I would like to be more generic.